packages/ui/src/council/modals/CandidacyPreview/CandidacyPreview.tsx

```diff
@@ -16,9 +23,16 @@ type ProfileTabs = 'CANDIDACY' | 'DETAILS' | 'ACCOUNTS' | 'ROLES'
 export const CandidacyPreview = React.memo(() => {
   const [activeTab, setActiveTab] = useState<ProfileTabs>('CANDIDACY')
   const { modalData } = useModal<CandidacyPreviewModalCall>()
-  const { isLoading, candidate } = useCandidate(modalData.id)
+  const [candidateId, setCandidateId] = useState(modalData.id)
+  const { isLoading, candidate } = useCandidate(candidateId)
+  const candidates = useElectionCandidatesIds(modalData.cycleId)
+  const candidateIndex = candidate && candidates?.findIndex((id) => id === candidate?.id)
+  const onClickLeft = () => candidates && isDefined(candidateIndex) && setCandidateId(candidates[candidateIndex - 1])
+  const onClickRight = () => candidates && isDefined(candidateIndex) && setCandidateId(candidates[candidateIndex + 1])
```

Comment:
Let's have a simple test for this modal:

*   Just 3 candidates
*   check states on first, last, and that left/right clicks changes candidate